client/src/components/RoleApproval.jsx

```diff
@@ -145,10 +145,7 @@ const RoleApproval = () => {
           }}
         />
       ) : (
-        <Card
-          className="interview-card"
-          style={{ height: '60%', margin: '2%' }}
-        >
+        <>
```

Comment:
Wait what was this lol (also if it's unneeded let's also fix that linting comment)

Reply:
The card was adding a border around the role approval when it was its own page but now there's an admin portal so...